src/resources/SecurityCache/tests/SecurityCache.spec.ts

```diff
@@ -145,4 +145,25 @@ describe('securityCache', () => {
             expect(api.get).toHaveBeenCalledWith(`${SecurityCache.cacheUrl}/status`);
         });
     });
+
+    describe('providerEntity', () => {
+        it('should make a POST call to the security Identity refresh url', () => {
```

Comment:
We might need to update this description, this call doesn't seem to point to the `refresh` URL.

Reply:
:scream: oh dear I totally forgot to change it